documentation/contributing/maintainer_pr/index.markdown

```diff
@@ -5,65 +5,84 @@ slug: maintainer_pr
-To ease the process of getting your contribution merged into the project, the following provides a guideline for maintainers.
+To ease the process of getting contributions merged into the project quickly, the following provides a guideline for maintainers.
 
-## Maintainer Side
+## Branches Policy
 
-The ``*-devel`` branches of the repositories must always compile on the target platforms.
+Our development branches are called ``main`` (MoveIt 2) and ``master`` (MoveIt 1). We also have important distribution-specific release branches, named ``*-devel`` (MoveIt 1) or just e.g. ``foxy`` (MoveIt 2).
 
-The ``*-devel`` branches must not be force-pushed.
+## No Direct Commits or Force Pushes
 
-### No Direct Commits
+No one is allowed to directly commit or force push to the development or distribution branches of the repositories. This should already be disabled in the Github repo settings.
 
-No one is allowed to directly commit to the ``*-devel`` branches of the repositories. Instead they should create feature branches and add pull-requests.
-The only commits that might be pushed directly are cherry-picks from older branches (see below) and administrative changes (e.g. CHANGELOGS, tags).
+The only commits that might be pushed directly are cherry-picks from older branches (see below) and administrative changes (e.g. CHANGELOGS, tags). However with Mergify bot this should be discouraged also.
 
-### Typically, two "Approves" == Merge
+## Feature Branches in Upstream Repositories
 
-If you see a trustworthy approval review in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
-merge the request after your own review. Otherwise submit an "Approval" review after you are satisfied after the review.
+Pull requests should originate from a user's fork of the main MoveIt upstream repositories as much as possible. Github has a feature that allows a MoveIt maintainer to contribute to a contributor's pull request branch [even if the branch is on another user's fork](https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/). Use of this is encouraged.
 
-For small or trivial fixes, a single review is sufficient. Additionally, for the ``moveit_experimental`` package, a single review is sufficient.
+### Exceptions to Feature Branches in Upstream Repos
 
-### GitHub Merge Policies
+In rare cases, if a maintainer expects support in the development of a patch from non-maintainer users, they are free to create a new branch in the upstream repository instead. This enables other maintainers to directly push changes there and enables users to add pull-requests targeting the feature branch. To keep the list of branches clear and unambiguous, **names for such branches should always follow the pattern ``pr-<ros distribution>-<keyword description>``**. This makes it clear these branches are not relevant to users not involved in the respective request.
```

Comment:
If it's a feature branch, I like `feature/description` more. At least that's what we've been using in the past.

Reply:
I did not change this policy, this is what is already on the website. I only reorganized its location.

I did think this was still our standard, though. I don't care either way.